spot-client/src/spot-tv/ui/components/meeting-frame/meeting-frame.js

```diff
@@ -240,6 +249,9 @@ export default class MeetingFrame extends React.Component {
      */
     _onMeetingLoaded() {
         this._meetingLoaded = true;
+        this.setState({
+            feedbackDisplayed: false
+        });
```

Comment:
Is false needed here?

Reply:
I don't was just doing it for safety if we'd be reloading it or something. I'm not sure if it makes sense ?